kuksa_databroker/databroker/src/main.rs

```diff
@@ -296,6 +299,41 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
             );
     }
 
+    #[cfg(feature = "viss")]
+    {
+        parser = parser
+            .arg(
+                Arg::new("enable-viss")
+                    .display_order(30)
+                    .long("enable-viss")
+                    .help("Enable VISSv2 service")
```

Comment:
Should we possibly mention "websocket", like "Enable VISSv2 websocket service", as VISS include specifications for HTTPS, Websocket and MQTT, but it is only websocket we support.

Reply:
done